package/src/components/Message/hooks/useMessageActions.tsx

```diff
@@ -291,9 +291,8 @@ export const useMessageActions = ({
   const retry: MessageActionType = {
     action: async () => {
       dismissOverlay();
-      const messageWithoutReservedFields = removeReservedFields(message);
```

Comment:
We already remove the reserved fields on the Channel component in `retrySendMessage` function so there's no point of doing it here as well.

Reply:
Can this be used without relying on `retrySendMessage` ? If so, we should keep it.